template-contributing-guide.md

```diff
@@ -0,0 +1,341 @@
+
+If you answered yes to more than one of these questions about a specific type of template, that might be the right template for you to work on!
+
+
+## Research and draft the template
+
+In this phase, you will research examples and identify best practices for the type of template you’re working on.
+At the end of this stage, our project recommends producing your working draft of the template project deliverables in a Google Doc. The reasons we recommend putting your draft in a Google Doc are because it:
+
+- Is free (no license required) and easy to use.
+- Is relatively easy to share with collaborators both inside and outside of the Good Docs project (such as with the Write the Docs community).
+- Allows collaborators to give feedback and advice in the form of comments.
+- Tracks comment history for later reference.
+- Has version control capabilities.
+
+That being said, you are not required to work in Google Docs. You can work in your preferred tool (such as Markdown, Microsoft Word, or Gitbooks). However, our experience has shown that Google Docs is the easiest tool for collaborating with other Good Docs Project contributors in the early phases of doc development.
+
+### Recommended research strategies
+
+In our experience, successful templateers usually research their template by:
+
+- **Looking at lots of examples.** Start by searching for examples of that type of document they want to create a template for. The more examples you can look at, the better. While it’s better to review good examples of that type of document, there is actually a lot of value in reviewing bad examples too. Consider keeping a spreadsheet to track which examples you used, what elements each one had in common, and what you thought was effective or ineffective.
+- **Searching for guides, books, blog posts, conference presentations, or videos about best practices.** Search the Internet to find advice, tips, or expert research about how to create that type of document. Consider posting in a forum for resource ideas. For example, asking for helpful guides or insights on a community forum like the Write the Docs Slack workspace could be beneficial. Be mindful of, and respect copyright terms of source material. Do not plagiarize and offer attribution where appropriate.
+- **Reaching out to experts.** When you find people you admire, who have researched your topic already, try reaching out to them. They often have a “how to contact me” webpage. Ask if they’d be okay with their material being used. (They might need to republish under a different copyright.) Invite them to participate in the template working group. They might even be prepared to lead it. If you feel shy about reaching out yourself, your template mentor or senior Good Docs Project member might offer to help.
+- **Collaborating with others in a working group.** Consider joining one of the template-focused working groups to meet regularly with a workshop of writers and discuss research ideas and findings.

```

Comment:
The phrasing here is "consider". But elsewhere it seemed more like a hard requirement that template authors join a template-writing working group. We should try to make more clear whether or not that is a hard requirement.

Reply:
It's a hard requirement and this reflects that our thinking about making it a requirement shifted over the course of drafting this guide. I'll update this to communicate that it's a hard requirement.

Reply:
I've adopted this suggestion.